root/static/scripts/edit/URLCleanup.js

```diff
@@ -1068,6 +1148,23 @@ const CLEANUPS: CleanupEntries = {
       return {result: false, target: ERROR_TARGETS.URL};
     },
   },
+  'bandcampmerch': {
+    match: [new RegExp(
+      '^(https?://)?([^/]+)\\.bandcamp\\.com/merch',
+      'i',
+    )],
+    restrict: [LINK_TYPES.mailorder],
+    clean: function (url) {
+      return url.replace(/^(?:https?:\/\/)?([^\/]+)\.bandcamp\.com\/merch\/([^\/]+)?.*$/, 'https://$1.bandcamp.com/merch/$2');
+    },
+    validate: function (url, id) {
+      switch (id) {
+        case LINK_TYPES.mailorder.release:
+          return {result: /^https:\/\/[^\/]+\.bandcamp\.com\/merch\/[\w-]+$/.test(url)};
+      }
+      return {result: false, target: ERROR_TARGETS.ENTITY};
+    },
+  },
```

Comment:
Any current example?

The test URL `https://non-serviam-records.bandcamp.com/merch/the-howling-void-megaliths-of-the-abyss-digipak` is not found.

Reply:
You know _NO FEAR_. Correct, I found https://web.archive.org/web/20191110201058/https://non-serviam-records.bandcamp.com/merch/the-howling-void-megaliths-of-the-abyss-digipak